### ISSUE TITLE
Is it a bug in file "PreprocessPDampingZones.f90"?

### ISSUE THREAD

**Issue by trimtrim** [apengsigkarup/OceanWave3D-Fortran90#5]
At  line 45 of PreprocessPDampingZones.f90, the nyd is computed by i3-i4+1, which may be a negative value. 
  **nyd=i3-i4+1**; PDampZones(i)%ny=nyd; nd=nxd*nyd

Does it need to change nyd = i4-i3+1??

**Comment by hbbingham** [comment]
Yes, that is indeed a bug and it should be i4-i3+1. Thanks for pointing that out.

Best regards,
Harry

**Comment by trimtrim** [comment]
Thanks for the confirmation. 
Best regards!
